app/containers/ReactionsList/calculatePadding.ts

```diff
@@ -0,0 +1,15 @@
+const HEADER_HEIGHT = 48;
+/**
+ * This SNAP_PERCENTAGE is calculate between the snaps: ['50%', '80%']
+ * so considering the content size is set to 80% and the first snap is 50%
+ * we need to 50% * 80% = 40%
+ */
+const SNAP_PERCENTAGE = 0.4;
```

Comment:
Similarly to my comment above, this is only used for this math here, but it's based on data coming from snaps.
What if we change the snaps?
Your code is going to fail.